letsencrypt-auto-source/letsencrypt-auto

```diff
@@ -367,6 +367,37 @@ BootstrapRpmCommon() {
   fi
 }
 
+BootstrapAlpineCommon() {
+  # Tested with:
+  #   - Alpine 3.4
+
+  $SUDO apk update || echo apk update hit problems but continuing anyway...
```

Comment:
Better use the `--no-cache` flag on the `apk add` commands below, to keep all garbageless.